mk/spksrc.common.mk

```diff
@@ -14,7 +14,7 @@ MSG = echo "===> "
 RUN = cd $(WORK_DIR)/$(PKG_DIR) && env $(ENV)
 
 # Pip command
-PIP ?= pip
+PIP ?= pip #define PIP in your makefile if you need pip3
 PIP_WHEEL = $(PIP) wheel --no-use-wheel --no-deps --wheel-dir $(STAGING_DIR)/wheelhouse
```

Comment:
isn't it a good practice to document such options? frankly speaking I find it not easy to quickly understand all compile options that spksrc offers.